r/accessing-and-managing-financial-data.qmd

```diff
@@ -118,40 +137,40 @@ factors_q_monthly_link <-
 
 factors_q_monthly <- read_csv(factors_q_monthly_link) |>
   mutate(date = ymd(str_c(year, month, "01", sep = "-"))) |>
-  select(-R_F, -R_MKT, -year) |>
-  rename_with(~ str_remove(., "R_")) |>
-  rename_with(~ str_to_lower(.)) |>
-  mutate(across(-date, ~ . / 100)) |>
+  rename_with(~sub("R_", "", ., fixed = TRUE)) |>
```

Comment:
I don't like this change particularly unless we explain briefly what is going on here. sub is way less obvious than str_to_lower, fixed = TRUE is completely opaque

Reply:
Agreed. I coped it from the package where it makes sense to not use stringr because of its crazy amount of dependencies.